authz/providers/azure/rbac/rbac.go

```diff
@@ -375,7 +378,94 @@ func (a *AccessInfo) CheckAccess(request *authzv1.SubjectAccessReviewSpec) (*aut
 
 		finalStatus = status
 	}
-	return finalStatus, nil
+	if finalStatus != nil && finalStatus.Allowed {
+		klog.V(5).Infof("Checkaccess request is allowed for user %s", checkAccessUsername)
+		return finalStatus, nil
+	}
+
+	if !a.useManagedNamespaceResourceScopeFormat || a.clusterType != managedClusters {
+		klog.V(5).Infof("Checkaccess request is denied for user %s", checkAccessUsername)
+		return finalStatus, nil
+	}
+
```

Comment:
is there a scenario where we can land in this part of the code for a managed cluster with `useManagedNamespaceResourceScopeFormat` false ?

Reply:
There shouldn't be, because of L386